### PR TITLE
Follow previous gitdm convention of having Heptio commits belong to VMware

### PR DESCRIPTION
**Pull request by castrojo**
There are a few exceptions like people who left prior to this, I am
leaving those untouched for now.

### CODE REVIEW

**Comment by lukaszgryglicki** [comment]
I will merge this tomorrow and this will come to the next prod version. It won't get into the test version because I've already importing this today.

File: developers_affiliations.txt [hunk 1]
```diff
@@ -9392,9 +9386,10 @@ Jochen Breuer: jbreuer!suse.de
 	SUSE
 Jodie Putrino: j.putrino!f5.com
 	F5Networks
-Joe Beda: joe!bedafamily.com, joe!heptio.com, joe.github!bedafamily.com
-	VMware
-	Heptio until 2018-12-11
+Joe Beda*: jbeda!vmware.com, joe!bedafamily.com
+    VMware
+	Google until 2016-12-01
+Joe Beda*: joe!heptio.com, joe.github!bedafamily.com
```
Comment (dankohn) [inline developers_affiliations.txt:9392]: This seems incorrect. Joe wasn't at CoreOS, was he?

File: developers_affiliations.txt [hunk 2]
```diff
@@ -1685,7 +1684,6 @@ Andy Diamondstein: adiamondstein!adiamondstein-macbookpro.roam.corp.google.com,
 	Google
 Andy Goldstein: agoldste!redhat.com, andy.goldstein!gmail.com
```
Comment (ncdc) [inline developers_affiliations.txt]: @castrojo please add `goldsteina@vmware.com` - thanks!

File: developers_affiliations.txt (hunk 1 above)
Comment (lukaszgryglicki) [reply developers_affiliations.txt:9392]: I will just wait with this a bit. Also if possible please fix the whitespace, we are using `tab` `\t` when adding company name. I can also fix that myself manually if this is a problem.

**Comment by lukaszgryglicki** [comment]
Also if we go that way, I can check ALL affiliations everywhere and replace `Heptio` with `VMware`.

File: developers_affiliations.txt [hunk 3]
```diff
@@ -13988,9 +13978,8 @@ NAVEEN NAHATA: naveen.nahata!blrmcb-78469.local
 	NotFound
 NIkhil Bhatia: nbhatia!microsoft.com
 	Microsoft
-Naadir Jeewa: naadir!randomvariable.co.uk
-	VMware
-	Heptio until 2018-12-11
+Naadir Jeewa: jeewan!vmware.com, naadir!randomvariable.co.uk
+    VMware
 	The Scale Factory until 2017-11-13
```
Comment (randomvariable) [inline developers_affiliations.txt]: I know not directly part of this PR, but this should be s/2017/2018

File: developers_affiliations.txt [hunk 4]
```diff
@@ -13988,9 +13978,8 @@ NAVEEN NAHATA: naveen.nahata!blrmcb-78469.local
...
 	uSwith until 2015-07-01
```
Comment (randomvariable) [inline developers_affiliations.txt]: Likewise, uSwith is actually uSwitch.

File: developers_affiliations.txt [hunk 5]
```diff
@@ -4058,7 +4055,7 @@ Chuanhong Wang: wang.chuanhong!zte.com.cn
 Chuanjian Wang: me!ckeyer.net
 	Sina
 Chuck Ha*: chuck!heptio.com, ha.chuck!gmail.com
-	Heptio
+	VMware
```
Comment (chuckha) [inline developers_affiliations.txt:4058]: thanks @castrojo ! lgtm

**Comment by castrojo** [comment]
@lukaszgryglicki A delay is actually useful as it's probably easier for me to just grab the corrections from everyone as they review, if you want to leave this open for a few days I can have people comment for corrections, then I'll just squash at the end of the week or something?

**Comment by lukaszgryglicki** [comment]
Let's not remove Heptio from `cncf/gitdm` config files, this is why:
- We will have a full history of all affiliations (richer data, including historical company names).
- We will not need to mass update company names in the future, which is error-prone.
- We already have a lot of data with past company names (pre-acquisition), like DEIS, Engine Yard (both are now Microsoft).
- When manually researching affiliations data, almost always we have historical company names. Most people will not update affiliations list to remove Heptio and replace with VMware, they will end Heptio on some date and then add VMware.
- I will add a special mapping in Devstats that will just read some JSON/YAML config that will map specific company names into other names.

That solves all problems IMHO. This PR can be merged manually then, only taking into account other stuff (not renaming `Heptio` to `VMware`).

**Comment by fabioy** [comment]
@castrojo Noticed that I hadn't updated my entry from my Google days (oops). The diff tool is not letting me add this comment (too many lines separating changes), but could you change it to:

```
Fabio Yeon: yeonf!vmware.com, fabioy!google.com, fabioy!users.noreply.github.com
        VMware
        Heptio until 2018-12-11
	Google until 2017-09-26
```

File: developers_affiliations.txt (hunk 1 above)
Comment (jbeda) [reply developers_affiliations.txt:9392]: Yeah -- I was an advisor there but was also doing a bunch of other stuff.  I'm happy to fix this stuff up in a separate PR.

**Comment by lukaszgryglicki** [comment]
Merged manually in a few last commits.

**Pull request closed without merging**